qemu/tests/cfg/virtio_fs_localfs_migration.cfg

```diff
@@ -0,0 +1,122 @@
+- virtio_fs_localfs_migration:
+    no RHEL.6 RHEL.7 RHEL.8.0 RHEL.8.1
+    no Win2008 Win7 Win2012 Win2012..r2 Win8 Win8.1
+
+    type = virtio_fs_localfs_migration
+    virt_test_type = qemu
+    required_qemu = [9.0.0,)
+    kill_vm = yes
+    start_vm = yes
+
+    filesystems = fs
+    filesystems_migration = targetfs
+    fs_driver = virtio-fs
+    fs_source_type = mount
+    fs_source_dir_fs = /var/tmp/virtio_fs_source
```

Comment:
All the fs paths are created by user, how about setting fs_source_user_config = yes, then it could save time in env_process.py

Reply:
marked this as resolved according to the discussion on #4196